Misc/NEWS.d/next/Library/2025-01-08-03-09-29.gh-issue-128562.Mlv-yO.rst

```diff
@@ -0,0 +1 @@
+Fix the bug that the widgets of :mod:`tkinter` are named inaccurately.
```

Comment:
Fix possible conflicts in generated :mod:`tkinter` widget names if the widget class name ends with a digit.